torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -361,6 +384,9 @@ std::vector<at::Tensor> FusionExecutorCache::runFusionWithInputs(
         // computational graph intact for future compilation.
         Fusion fusion_clone = *fusion_;
         FusionGuard fg(&fusion_clone);
+        for (auto out : fusion_clone.outputs()) {
+          std::cout << out << std::endl;
+        }
```

Comment:
Is this just for debugging?

Reply:
Yes, removed it. Thanks for pointing this out!